### PR TITLE
Fix primary owner transfer not being set to OWNER role

### PR DESCRIPTION
**Pull request by Diet-Cola**

### CODE REVIEW

File: plugins/namelayer-paper/src/main/java/vg/civcraft/mc/namelayer/command/commands/TransferGroup.java [hunk 1]
```diff
@@ -66,9 +66,11 @@ public static boolean attemptTransfer(Group g, Player owner, UUID futureOwner) {
 					+ " is not a member of the group and can't be made primary owner!");
 			return false;
 		}
+		g.removeMember(futureOwner);
+		g.addMember(futureOwner, GroupManager.PlayerType.OWNER);
```
Comment (Protonull) [inline plugins/namelayer-paper/src/main/java/vg/civcraft/mc/namelayer/command/commands/TransferGroup.java:70]: Just thought I'd put this here :P 
image

Comment (Diet-Cola) [reply plugins/namelayer-paper/src/main/java/vg/civcraft/mc/namelayer/command/commands/TransferGroup.java:70]: Best part about this is, I followed the logic of the promote player command... which doesnt even use this either

Comment (Protonull) [reply plugins/namelayer-paper/src/main/java/vg/civcraft/mc/namelayer/command/commands/TransferGroup.java:70]: Civ code D:

**Comment by Diet-Cola** [comment]
Fixes #253

**Pull request merged**